spec/controllers/problem_reports_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProblemReportsController, type: :controller do
+  describe 'GET problem_reports#index' do
+    before do
+      sign_in create(:user)
+      get :index
+    end
+
+    it { should respond_with :ok }
+  end
+
+  describe 'POST problem_reports#create' do
```

Comment:
You also need to check if the model object is actually created

Reply:
added